.github/workflows/add-preview-url.yml

```diff
@@ -14,6 +14,13 @@ jobs:
           github-token: ${{ secrets.LOCULUS_BOT_WRITE_PRS }}
           script: |
             const pr = context.payload.pull_request;
+
+            // Skip if PR author is Dependabot for security reasons
```

Comment:
Could you explain a bit more why we need this? You still have to add a label – all the action does is to add a link?

Reply:
Ok, I understand now the check was failing. I would delete "security reasons" and say "because it doesn't work" or something but nbd

Reply:
ah so it doesnt work because we cant give dependabot the keys due to how github enforces key access - I will try to make this clearer

Reply:
In fact we could also just add this as a dependabot secret

![image](https://github.com/user-attachments/assets/04d178e8-45fd-42f9-829a-441e92d234ed)

but I've probably lost the secret now so this may be easier for now